package.json

```diff
@@ -11,19 +11,12 @@
     "clean": "shx rm -rf public"
   },
   "devDependencies": {
-    "@tailwindcss/line-clamp": "^0.4.0",
-    "concurrently": "^6.2.2",
-    "cross-env": "^7.0.3",
-    "hugo-extended": "^0.89.4",
-    "shx": "^0.3.4",
-    "tailwindcss": "^2.2.16"
+    "concurrently": "~7.6.0",
+    "cross-env": "~7.0.3",
+    "shx": "~0.3.4"
   },
   "workspaces": [
     "hugo",
     "tailwind"
-  ],
-  "volta": {
```

Comment:
I think we want to keep the volta config if we can. That has helped a lot for isolating env related issues.